packages/keplr/src/signer.ts

```diff
@@ -0,0 +1,158 @@
+import { AccountData, DirectSignResponse } from "@cosmjs/proto-signing";
+import {
+  ChainInfo,
+  Keplr,
+  Window as KeplrWindow,
+  KeplrSignOptions,
+} from "@keplr-wallet/types";
+import { SignDoc } from "cosmjs-types/cosmos/tx/v1beta1/tx";
+import { AminoSignResponse, StdSignDoc, Algo } from "@cosmjs/amino";
+import { assert } from "@cosmjs/utils";
+import { Signer, SignerStatus, SigningMode } from "@desmoslabs/desmjs";
+import { DesmosMainnet } from "./chains";
+
+declare global {
+  interface Window extends KeplrWindow {}
+}
```

Comment:
What is this used for? Can we put a comment here so that we remember in the future why this code is here?

Reply:
Sure, added. It just adds Keplr Typescript definitions to the window interface